### ISSUE TITLE
In variable pricing Add New Price should be a button

### ISSUE THREAD

**Issue by samikeijonen** [easydigitaldownloads/easy-digital-downloads#4344]
Check issue #4343 for reference. At the moment **Add New Price** is a link.

![add_new_price](https://cloud.githubusercontent.com/assets/1820415/13254072/42ab6624-da48-11e5-83c4-f97c3d141572.png)

It's not focusable using keyboard only. I suggest that we change that to button as well.

````
<button class="button-secondary edd_add_repeatable" style="margin: 6px 0;"><?php _e( 'Add New Price', 'easy-digital-downloads' ); ?></button>
```

**Comment by samikeijonen** [comment]
The same should be done in Download Files metabox for Add New File button.

**Comment by pippinsplugins** [comment]
:+1:

**Comment by pippinsplugins** [comment]
PR works great, merged into release/2.6!
